packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -117,31 +69,7 @@ export const HelpCenterMoreResources = () => {
 						</a>
 					</div>
 				</li>
-				{ showWhatsNewItem && (
-					<li className="help-center-more-resources__resource-item help-center-link__item">
-						<div className="help-center-more-resources__resource-cell help-center-link__cell">
-							<Button
-								variant="link"
-								onClick={ handleWhatsNewClick }
-								className="help-center-more-resources__new-releases"
```

Comment:
There are some unused styles to be removed related to this no-longer-used classname.